modin/pandas/test/utils.py

```diff
@@ -431,6 +431,9 @@
 # the type of this exceptions are the same
 io_ops_bad_exc = [TypeError, FileNotFoundError]
 
+# Files compression to extension mapping
+comp_to_ext = {"gzip": "gz", "bz2": "bz2", "xz": "xz", "zip": "zip"}
+
```

Comment:
Move into `_csv_file_maker`. The context is too big for this variable considering how it is used.

Reply:
`comp_to_ext` is also needed in `test_read_csv_compression` function, so it was moved to the `test_io.py`.